lib/src/pdf_page_proxy.dart

```diff
@@ -18,33 +18,48 @@ class PDFPageProxy {
   JsObject _jsInternal;
 
   PDFPageProxy() {
-    _jsInternal = JsObject(context['pdfjsLib']['PDFPageProxy']);
+    checkNotNull(context, message: 'context was null in PDFPageProxy');
+    final pdfjsLib = context['pdfjsLib'];
+    checkState(pdfjsLib != null, message: 'pdfjsLib was null in PDFPageProxy');
+    final pdfPageProxyJs = pdfjsLib['PDFPageProxy'];
+    checkState(pdfPageProxyJs != null, message: 'pdfPageProxyJs was null in PDFPageProxy');
```

Comment:
is there a difference between `checkNotNull` and `checkState(something != null`?

Reply:
Not entirely sure. This is "practically" a straight port over of what is in markup_client today (and I presume [exactly what is in v2.5.207](https://github.com/Workiva/markup_client/tree/master/lib/src/static/pdfjs/src) of the mozilla pdf.js interop code). Only change we made was to address a bug we discovered during dev.

Reply:
I'd like to try to keep it as close to the mozilla version as possible

Reply:
Makes sense. Maybe once we've consolidated we can come back and address this. But I'm sure the dart packager is doing tree shaking and quiver is owned by google so probably dont need to worry so much about a leftpad type situation